### DISCUSSION TITLE
Darstellungsfehler im Dashboard?

### DISCUSSION THREAD

**Discussion by Sprudelverduenner** [dewenni/ESP_Buderus_KM271#41] [General]
Hallo Sven,

ich habe es nun geschafft in FHEM per MQTT auch die Befehle, die ich brauche,  zu integrieren.

Dabei ist mir gerade wahrscheinlich ein Darstellungsfehler im Dashboard aufgefallen.

Sobald ich bei HK1 die Betriebsart von AUTOMATIK auf TAG oder NACHT stelle dann wird bei HK1 Sommer/Winter IMMER WINTER angezeigt.
Bei den Daten zu Heizkreis 1 wird aber richtigerweise bei HK1_Sommer_ab SOMMER angezeigt.

Andere ich bei HK1 die Betriebsart wieder auf AUTOMATIK dann stimmt die Information bei HK1 Sommer/Winter wieder im Dashboard.

Ich hoffe ich habe mich verständlich ausgedrückt... :-D

Beste Grüße und danke für diese großartige Software,
Uwe

**Comment by dewenni** [comment]
Hallo Uwe,

danke für das Feedback. Das scheint tatsächlich falsch zu sein.

Der Status im Dashboard für Sommer/Winter wird von den Betriebswerten 2 des HK1 abgeleitet.
Die Betriebswerte sind nach dieser [Vorlage](https://github.com/mhop/fhem-mirror/blob/28553ace9f7373517b77eccaf429ad8bbaa1b760/fhem/FHEM/00_KM271.pm#L240) ein Bit-Array.
Vom ersten Bit habe ich dann den Status abgeleitet. 1=Sommer 0=Winter
Das funktioniert und stimmt auch für den Automatikmodus.
Im manuellen Modus (Tag/Nacht) gilt das aber wohl nicht. Da gibt es vermutlich gar keine Sommer/Winter Umschaltung.

Ic habe das gerade auch mal an der Heizung selbst kontrolliert. Wenn man die Betriebsart von Automatik auf z.B. Tag stellt, dann verschwindet auch an der Heizung das Zeichen für Sommer 🌴 im Display der Heizung.

Ich müsste also den Status an die Betriebsart koppeln und im manuellen Modus (Tag/Nacht) dann auch bei Sommer/Winter als Status "Manuell" oder "---" ausgeben.

Das werde ich im nächsten Release einfließen lassen.

**Reply by Sprudelverduenner** [reply]
Wobei im Reiter Heizkreis es meiner Meinung nach richtig angezeigt wird…

**Reply by dewenni** [reply]
`HK1_Sommer_ab` gibt ja nur die Umschaltschwelle an. Entweder eine Temperatur zwischen 10..30°C oder fix auf SOMMER oder Winter. Das ist also nur ein Config Wert.

Abhängig von der Einstellung unter "Bedienung => HK1_Sommer_ab"

**Comment by dewenni** [comment]
Korrektur mit Release v3.2.4
